src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/WebSocketsTransport.cs

```diff
@@ -180,6 +180,13 @@ public WebSocketsTransport(WebSocketOptions options, Channel<byte[]> application
 
                             await ws.SendAsync(new ArraySegment<byte>(buffer), _options.WebSocketMessageType, endOfMessage: true, cancellationToken: CancellationToken.None);
                         }
+                        catch (WebSocketException socketException)
+                        when (socketException.WebSocketErrorCode == WebSocketError.InvalidState)
```

Comment:
We never throw here for any exception, we just log it and exit. I wasn't sure what states were relevant
```
None = 0,
Connecting = 1,
Open = 2,
CloseSent = 3,
CloseReceived = 4,
Closed = 5,
Aborted = 6
```

I think `Closed Aborted CloseSent` would be the ones to cause the offending exception.